pythonAPIClient/client.py

```diff
@@ -25,11 +28,11 @@ def header_builder(self):
         return head
 
     def __init__(self, id, secret):
-        #clientId are of a fixed length of 96 characters.
+        # clientId are of a fixed length of 96 characters.
         if len(id) != 96:
             raise InvalidClientIdError
         # clientSecret are of a fixed length of 71 characters.
```

Comment:
clientsecret  here should be 72 too.
I was thinking instead of hardcoding numbers we do something like lengthOfSecret=72
len(secret)!= lengthOfSecret

Reply:
krishna it's 71.